### PR TITLE
fix(RfiFrameDrop): mutex can't get acquired on second frame

### PR DESCRIPTION
**Pull request by nritsche**
There was a typo. Changed lock.release() -> lock.unlock()
release() releases the ownership over the mutex.

### CODE REVIEW

File: lib/stages/RfiFrameDrop.cpp [hunk 1]
```diff
@@ -136,7 +140,7 @@ void RfiFrameDrop::main_thread() {
                                  / (4.0 * sk_step * sk_step));
 
         // Lock update mutex to not allow updates being processed during this critical section
-        std::unique_lock<std::mutex> lock(update_mutex);
+        lock.lock();
```
Comment (jrs65) [inline lib/stages/RfiFrameDrop.cpp:143]: Why not just put this section into a block and use a lock_guard? Seems more elegant.

**Review by jrs65 — APPROVED** [review]
One mild suggestion, but otherwise approved.

**Review by james-s-willis — APPROVED** [review]
Changes look good to me

File: lib/stages/RfiFrameDrop.cpp (hunk 1 above)
Comment (nritsche) [reply lib/stages/RfiFrameDrop.cpp:143]: Variable scope. I would have to declare them outside of the block and initialize inside.

**Pull request merged**